src/content/popup/Words/WordTable.fixture.tsx

```diff
@@ -0,0 +1,233 @@
+import { WordResult } from '@birchill/jpdict-idb';
+import { useFixtureInput, useFixtureSelect } from 'react-cosmos/client';
+
+import {
+  AccentDisplay,
+  PartOfSpeechDisplay,
+} from '../../../common/content-config-params';
+
+import { ShowPopupOptions } from '../show-popup';
+
+import { WordTable } from './WordTable';
+
+const options: Omit<
```

Comment:
(I think if we can trim the set of options passed to `WordTable` we could make this type a bit smaller?)